test.js

```diff
@@ -52,6 +64,10 @@ test('hash with argon2d', async t => {
   t.is(await argon2.hash(password, salt, {type: argon2.argon2d}), hashes.argon2d)
 })
 
+test('argon2d with raw hash', async t => {
+  t.is((await argon2.hash(password, salt, {type: argon2.argon2d, raw: true})).equals(hashes.rawArgon2d), true)
```

Comment:
As far as I know we cannot test the buffer's equality with the === operator. The parenthesis are there, because we have to test that on the buffer not on the Promise<Buffer>